test/functional/eval/system_spec.lua

```diff
@@ -446,8 +446,11 @@ describe('systemlist()', function()
     after_each(delete_file(fname))
 
     it('replaces NULs by newline characters', function()
-      if helpers.pending_win32(pending) then return end
-      eq({'part1\npart2\npart3'}, eval('systemlist("cat '..fname..'")'))
+      if iswin() then
+        eq({'part1\npart2\npart3\r'}, eval('systemlist("type '..fname..'")'))
```

Comment:
can't we use `cat` on windows?

Reply:
Yes, but the trailing `\r` remains. See https://github.com/neovim/neovim/pull/7343#discussion_r143318068.

Reply:
Sure, but the more we can do to avoid platform-specific variations, the better.